### PR TITLE
LPS-146069 Move xstream.jar and its dependencies

### PR DESCRIPTION
**Pull request by rafaprax**

### CODE REVIEW

**Comment by rafaprax** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:test

File: modules/dxp/apps/portal-rules-engine-drools/portal-rules-engine-drools/build.gradle [hunk 1]
```diff
@@ -9,7 +9,6 @@ dependencies {
 	compileInclude group: "org.mvel", name: "mvel2", version: "2.1.0.drools16"
 
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
-	compileOnly group: "com.thoughtworks.xstream", name: "xstream", version: "1.4.17"
```
Comment (tinatian) [inline modules/dxp/apps/portal-rules-engine-drools/portal-rules-engine-drools/build.gradle:12]: @rafaprax Why you remove this? If there is a reference to the classes of "com.thoughtworks.xstream" inside this module, we should update this, if not, this is not needed even if you did not remove those jars in portal/lib, then you should do this related clean up first.

File: modules/dxp/apps/portal-rules-engine-drools/portal-rules-engine-drools/bnd.bnd [hunk 2]
```diff
@@ -3,35 +3,22 @@ Bundle-SymbolicName: com.liferay.portal.rules.engine.drools
 Bundle-Version: 5.0.9
 DynamicImport-Package: *
 Import-Package:\
-	!com.bea.*,\
```
Comment (tinatian) [inline modules/dxp/apps/portal-rules-engine-drools/portal-rules-engine-drools/bnd.bnd:6]: @rafaprax The commit message "Use exported xstream api" is confusing to me. So you add a separate bundle to expose "com.thoughtworks.xstream" packages, but the removed packages here are not the packages you exposed. 

If the lines you removed are actually useless, then please do a clean up before you made your changes.

Tina.

**Comment by rafaprax** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
Running test suites again because I included one more related jar.

**Comment by rafaprax** [comment]
ci:test

**Comment by vicnate5** [comment]
@tinatian No regressions found

**Comment by tinatian** [comment]
@shuyangzhou This is good to go.

Tina

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/113224

**Comment by shuyangzhou** [comment]
ci:close

**Pull request closed without merging**